client/src/components/modals_and_popovers/FocusLockedModal.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { FocusLockedModal } from "./FocusLockedModal";
+
+export default {
+  title: "FocusLockedModal",
+  component: FocusLockedModal,
+};
+
+const Template = (args) => <FocusLockedModal {...args} />;
```

Comment:
Story doesn't demonstrate the "focus locked" aspect of this component, let me know if you need a tip on how to do that.

Reply:
I think my updated version might demonstrate it a bit better.